src/app/services/doc.service.ts

```diff
@@ -8,33 +8,30 @@ import * as data from '../../git-version.json';
 export class DocService {
   public gitDistance = data['default'].distance;
   public appVersion = data['default'].tag;
-
-  constructor() { }
+  public version = 'develop';
+  constructor() {
+    this.version = this?.gitDistance > 0 ? 'develop' : `${this?.appVersion}`;
```

Comment:
do we really need string interpolation here? and `this?`

Reply:
I suggest to make this

```
  private version = "develop"; //readthedocs version to use

  constructor() {
    if (this.gitDistance == 0) {
      this.version = this.appVersion;
    }
  }
```
so that in case a custom branch doc tests needed (say 2.0.1RC), GUI can be tested with single change